### ISSUE TITLE
Course reserves cleanup

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#558]
- [x] **All fields dropdown for the main search box is missing.**
  ![screen shot 2014-07-24 at 1 46 36 pm](https://cloud.githubusercontent.com/assets/6945691/3694558/a5adfe96-1373-11e4-8582-e1e86a77cf73.png)
- [x] Remove the left-right padding so the table visually aligns with the left and right margins.
- [x] Show 25 per page by default
- [x] Remove the colon from the datatable search box label.
- [x] Modify the wording of the top controls as shown
- [x] Add the same background as the results and record view toolbars

```
background-color: #f2f1eb;
border-bottom: 3px solid #d5d0c0;
```

![screen shot 2014-07-24 at 1 38 58 pm](https://cloud.githubusercontent.com/assets/6945691/3694538/6826e344-1373-11e4-8f38-7aa527a63602.png)
- [x] Make the padding above and below the pagination match the regular search results page. (For me this was at 38px)
- [x] IDEALLY use the same pagination control as on regular results page (with prev/next at the front)
- [x] Alternatively... can we at least make the arrow icons match?
- [x] and the active page should not be a link, but a span (as with default pagination control)
- [x] but if it has to remain a link, then the :hover colour should match the .active colour (dimgrey)

![screen shot 2014-07-24 at 1 40 45 pm](https://cloud.githubusercontent.com/assets/6945691/3694542/6e06b55a-1373-11e4-811c-b6f6e7796969.png)
